### ISSUE TITLE
Migration from 1.16.2 to 1.17.0 fails

### ISSUE THREAD

**Issue by jjettenn** [molgenis/molgenis#4587]
Updating molgenis39 from 1.16.2 to 1.17.0 resulted in errors:

```
Caused by: java.lang.NullPointerException: null
	at org.molgenis.migrate.version.v1_17.Step27MetaDataAttributeRoles$3.processRow(Step27MetaDataAttributeRoles.java:127) ~[molgenis-data-migrate-1.17.0.jar:1.17.0]
```

Expected the migration to succeed.

**Comment by tommydeboer** [comment]
Can not reproduce. I think we need (a subset of) the real data to see wat goes wrong.

**Comment by jjettenn** [comment]
Will be fixed by: #4596 Uploading 10001 rows (data) via the rest API will result in a Mysql backend lock
